### ISSUE TITLE
[0.49.14 Nightly] Unusually long pause between turn phases

### ISSUE THREAD

**Issue by PhoenixHeart512** [MegaMek/megamek#4557]
### Environment
megamek.MegaMek.initializeLogging(MegaMek.java:116) - Starting MegaMek v0.49.14-SNAPSHOT
	Build Date: 2023-06-19T01:28:48.912615100
	Today: 2023-06-23
	Origin Project: MegaMekLab
	Java Vendor: Eclipse Adoptium
	Java Version: 17.0.6
	Platform: Windows 10 10.0 (amd64)
	System Locale: en_US
	Total memory available to MegaMek: 4 GB

### Description
This will be several bug reports using the same logs and files as there are multiple errors showing in the log that should be tracked separately.

As the game progresses, the time it takes to resolve each phase gets longer and longer, and by turn 8 or so it was taking 10-15 seconds to display the report for the next phase, as if the game is getting bogged down more and more each turn. I've noticed this same behavior on ground battles as well, this is not specific to this Air Support mission.

This may be related to, or caused by, all of the various errors in the attached log. The game I played yesterday, by the time I was finished the game the megamek.log file was 13MB in size. Unforutnately I didn't think to copy the file out before I finished and it wiped the file upon scenario completion, so I do not have logs for that game, only the autosaves (also attached)

### Files
I've included all autosaves from the game so that they can be loaded at different points in the game as needed
[Log Errors and Long Turns.zip](https://github.com/MegaMek/megamek/files/11851139/Log.Errors.and.Long.Turns.zip)

**Comment by SJuliez** [comment]
FWIW: I played a game of 1 unit vs 1 unit against Princess as a test yesterday and also found that the time between phases was unusually long and seemed to increase. I had no unexpected messages in the log. Something seems to accumulate

**Comment by BLR-IIC** [comment]
I've experienced this as well.  Phase resolution definitely bogs down with a gray blank report window in between.

**Comment by Duuvian** [comment]
I also ran into this. I have an autosave and campaign file (from a daily build a while ago) but apparently I too forgot to save the logs.

IIRCI think I was unsure if it was my low quality hard drive's fault (the Microsoft virus scanner takes it to 50% usage and it takes a good 10 seconds to open a firefox window like that...) or if it was a MHQ thing. Overall MHQ is running great with the excellent optimaztion work when compared to the old days, though whereas the previously time consuming move calculations are running smooth and fast, the end turn hangs draw out (in my case bot controlled) player team scenario into taking a really, really long time. A large battle with 30ish units can finish it's moves faster than the end turn dialogue proceeds (after 10-15 rounds)  for example, by a wide margin.

**Comment by Thom293** [comment]
Me too.  Its enough to be annoying.  Saves and logs attached.
[PhaseLag.zip](https://github.com/MegaMek/megamek/files/12222785/PhaseLag.zip)

**Comment by HammerGS** [comment]
This might be linked to the unit tooltips in the round report. We’ve had reports that turning them off resolves the lag.

**Comment by Thom293** [comment]
Yeah, I turned off High Quality Graphics to test, and that seemed to help a little.  But turning off unit cards in the report removed the lag completely.  

During discussions, it was also discussed that the option to turn off the unit cards in the reports should be moved to the report tab under client settings, instead of game options where it is at now.  Its the only report rellated option there.  I could not find it at first.  Would seem it would belong under the unit sprite option on the reports tab.

**Comment by AQGadget** [comment]
Turning off the sprites in the reports greatly improved the lag between phases

**Comment by PhoenixHeart512** [comment]
@HammerGS This issue can be closed, it was resolved as a side effect of other changes made during development and no longer occurs
